cpp/include/torch_tensorrt/ptq.h

```diff
@@ -314,7 +315,7 @@ class Int8CacheCalibrator : Algorithm {
  */
 
 template <typename Algorithm = nvinfer1::IInt8EntropyCalibrator2, typename DataLoader>
-TORCHTRT_API inline Int8Calibrator<Algorithm, DataLoader> make_int8_calibrator(
+inline Int8Calibrator<Algorithm, DataLoader> make_int8_calibrator(
```

Comment:
Do these need to be exported as `TORCHTRT_API`? Seems like in #1091 they are not being exported properly

Reply:
I _think_ these do not need to be exported, since the implementation is in the header, so it's not that the symbols need to be found in a dll... 
I'll try to investigate a bit more and comment on the issue #1091 directly.

Reply:
For the record, I just commented on that issue.